mithril.js

```diff
@@ -28,6 +28,17 @@
 		return "v0.2.5"
 	}
 
+	var div = document.createElement("div")
+	var frag = document.createDocumentFragment()
+
+	function toDOM(HTML){
+		div.innerHTML = HTML
+
+		for (var i = 0; i < div.childNodes; i++) {
+			frag.appendChild(div.childNodes[i])
+		}
```

Comment:
Looks like this function is missing a return value?

Or maybe even:

    var div = document.createElement("div")

    function toDOM(HTML){
        var frag = document.createDocumentFragment() // ensure it's an empty fragment each time
        div.innerHTML = HTML

        for (var i = 0; i < div.childNodes; i++) {
            frag.appendChild(div.childNodes[i])
        }
        return frag
    }

(Was just looking at this out of curiosity, and it kinda jumped out -- feel free to ignore me :-) )